src/components/Firestore/Requests/index.scss

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.Firestore-Request-Date {
+  font-size: 13px;
+  font-family: Courier;
+  font-style: normal;
+  font-weight: normal;
+  color: var(--mdc-theme-text-black-secondary);
+}
+.Firestore-Request-Outcome .Firestore-Request-Outcome-Icon {
+  color: var(--mdc-theme-primary);
+  // Center icon
+  display: block;
+  margin: 0 auto;
+  text-align: center;
+}
+.Firestore-Request-Method {
+  text-transform: uppercase;
+  font-size: 13px;
+  font-family: Courier;
```

Comment:
Do we have a fallback to "monospace" elsewhere or is it all just Courier (@yuchenshi )

Reply:
I updated the font-family to **Courier** to match the style of the [Figma UX visual feedback](https://www.figma.com/file/oz3FtG70f3xfYVado4Qvha/Request-Viewer-Feedback?node-id=2%3A51) from marissa, but you folks let me know if I should change it.

![image](https://user-images.githubusercontent.com/34299428/105211953-76774900-5b12-11eb-98e4-4d0eede71d4e.png)

Reply:
Let's be consistent with other existing monospace fonts in this project. Please take a look -- I believe `Roboto Mono` is used with some fallbacks

Reply:
+1 to Yuchen's response. This is what I mean by fallback: https://stackoverflow.com/questions/4673963/css-fallback-fonts

Reply:
I see, it makes sense. I took a look at other font fallbacks in the project and they indeed use `'Roboto Mono', 'monospace'`, so I will use: `font-family: Courier, 'Roboto Mono', monospace;`

Reply:
After quick font discussion on gChat, we decided to get **rid of Courier** and just go with `font-family: 'Roboto Mono', monospace;` instead.

![image](https://user-images.githubusercontent.com/34299428/105218782-349ed080-5b1b-11eb-9763-fe50b7bced5a.png)